.dockerfile

```
@@ -0,0 +1,14 @@
+FROM python:3.10.0
+
+LABEL maintainer="Gorgeous-Ghouls"
+
+RUN pip install poetry
+
+RUN git clone https://github.com/Gorgeous-Ghouls/Blak.git /home/Blak
```

```suggestion
RUN mkdir /server
WORKDIR /server
COPY . .
```
as the `working_directory` in GH action will be `server/`